lib/menu.rb

```diff
@@ -0,0 +1,42 @@
+class Menu
+
+  attr_reader :menu
+
+  def initialize
+    @menu = {}
+  end
+
+  def make_dish_available(dish)
+    number = next_dish_number
+    @menu[number] = dish
+  end
+
+  def view
+    fail "No dishes available on this menu" if no_dishes_available?
+    menu.each do |k, v|
+      puts "#{k}: #{v.name} | Price: £#{'%.2f' % v.price}"
```

Comment:
Style/FormatString: Favor `format` over `String#%`.<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.